src/widgets/merch/ui/merch.module.scss

```diff
@@ -10,10 +10,11 @@
     }
   }
 
-  .picture {
+  .image {
     width: 680px;
     height: 700px;
     margin: 0 auto;
+    object-fit: cover;
 
     @include media-laptop {
```

Comment:
```
@include media-laptop {
    flex-direction: column;
  }
```
This part is unnecessary because the `.column-2` class has the same behavior added to it

Reply:
Done